creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -117,7 +119,61 @@ const healthCheck = async ({ libs, snapbackSM } = {}, logger, sequelize, getMoni
     snapbackModuloBase,
     snapbackJobInterval,
     transcodeActive,
-    transcodeWaiting
+    transcodeWaiting,
+    fileProcessingActive,
+    fileProcessingWaiting
+  }
+
+  // If optional `randomBytesToSign` query param provided, node will include string in signed object
```

Comment:
not sure if this is a bad merge but this looks like it's duplicated?

Reply:
oops yeah fixed it